JetStreamCompose/jetstream/build.gradle.kts

```diff
@@ -14,91 +14,82 @@
  * limitations under the License.
  */
 
-import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
-
+@Suppress("DSL_SCOPE_VIOLATION") // TODO: Remove once KTIJ-19369 is fixed
 plugins {
     alias(libs.plugins.android.application)
     alias(libs.plugins.kotlin.android)
     alias(libs.plugins.kotlin.serialization)
-    id 'kotlin-kapt'
+    kotlin("kapt")
```

Comment:
could we use `ksp` instead of `kapt`? I assume it's for Hilt, which added support in 2.48 https://dagger.dev/dev-guide/ksp.html

Reply:
Thanks for the suggestion, and I am thinking I will make the change to ksp in another PR as I would make this PR for migration to the kts file.